aws/routeAwsGet.go

```diff
@@ -26,15 +26,40 @@ import (
 	"github.com/trackit/trackit2/users"
 )
 
+// uintSliceContainsUint returns true if a slice of uint contains id.
+func uintSliceContainsUint(slice []uint, id uint) bool {
+	for _, idd := range slice {
+		if idd == id {
+			return true
+		}
+	}
+	return false
+}
+
+// stripAwsAccounts strips the slice of AwsAccount with filters in the
+// arguments map with QueryArgAwsAccounts as the key.
+func stripAwsAccounts(awsAccounts []AwsAccount, a routes.Arguments) []AwsAccount {
```

Comment:
With this method, you still get every account from the database, only to drop them. Please implement this feature with database queries instead. In SQL you can select `WHERE value IN (42, 32, 10);` (or some similar syntax).